zproject/urls.py

```diff
@@ -158,6 +158,13 @@
     # Returns a 204, used by desktop app to verify connectivity status
     url(r'generate_204$', zerver.views.generate_204, name='zerver.views.generate_204'),
 
+    # realm/aliases -> zerver.views.realm_aliases
+    url(r'^realm/aliases$', rest_dispatch,
+        {'GET': 'zerver.views.realm_aliases.list_aliases',
+         'POST': 'zerver.views.realm_aliases.create_alias'}),
+    url(r'^realm/aliases/(?P<alias_id>\d+)$', rest_dispatch,
+        {'DELETE': 'zerver.views.realm_aliases.delete_alias'}),
```

Comment:
Can you change these to `/realm/domains`?  We're planning to rename the RealmAlias model to RealmDomain, and we might as well make the API use the new names to avoid having to migrate it later :).

I guess that probably entails a bunch of other changes to this PR as well, but I think it's worth doing.